src/api/app/assets/javascripts/webui2/staging_workflow.js

```diff
@@ -1,11 +1,9 @@
 function setSpinnersForDeletion() { // jshint ignore:line
-  $(".delete-spinner").hide();
-
-  $(document).ajaxStart(function() {
-    $(".delete-spinner").show();
+  $("#staging-workflow-delete").on('ajax:beforeSend', function(){
+    $(this).find('.delete-spinner').removeClass('d-none');
   });
 
-  $(document).ajaxStop(function() {
-    $(".delete-spinner").hide();
+  $("#staging-workflow-delete").on('ajax:complete', function(){
+    $(this).find('.delete-spinner').addClass('d-none');
```

Comment:
why? I don't get it - $(".delete-spinner").hide(); is just perfectly fine.

@bgeuken ?

Reply:
@coolo i think @bgeuken intention is that in case we are enhancing the view with some more ajax based stuff (we dont know yet about), that is not related to the form, the spinner would get triggered through it as well.

Reply:
@krauselukas so why call it delete-spinner then if it's potentially for something else? :)

And d-none is most likely the wrong tool to hide a spinner. Better use https://getbootstrap.com/docs/4.0/utilities/visibility/